qcfractal/interface/models/common_models.py

```diff
@@ -39,8 +39,8 @@ def validate(cls, v):
 
 
 class DriverEnum(str, Enum):
-    """The possible driver configurations of a single quantum chemistry
-    computation.
+    """
+    The driver configuration of a single quantum chemistry computation. This is a finite set of choices.
```

Comment:
Can you use the string that I wrote for this verbatim? "The type of calculation that is being performed (e.g., energy, gradient, Hessian, ...)."